### PR TITLE
Feature/acms 1556/add eu support for Moodle

### PR DESCRIPTION
**Pull request by naveenbalachandar**
This PR adds support for EU server in Moodle
1) Added  new admin setting checkbox - is_eu
2) Set api_endpoint to "https://eu.api.accredible.com/v1/" if the is_eu flag is set

Here is how the settings page will look like now.
Screenshot 2021-06-14 at 6 28 35 PM

### CODE REVIEW

File: lang/en/accredible.php [hunk 1]
```diff
@@ -32,6 +32,8 @@
 $string['autoissueheader'] = 'Automatic issuing criteria';
 $string['apikeylabel'] = 'API key';
 $string['apikeyhelp'] = 'Enter your API key from accredible.com';
+$string['eulabel'] = 'Europe server';
```
Comment (alanheppenstall) [inline lang/en/accredible.php:35]: Can we change to EU (Frankfurt) Server?

File: lang/en/accredible.php [hunk 2]
```diff
@@ -32,6 +32,8 @@
...
+$string['euhelp'] = 'Choose if you want to use the EU server';
```
Comment (alanheppenstall) [inline lang/en/accredible.php:36]: Select if you would like to host data in the EU (Frankfurt) instead of the USA.

**Review by alanheppenstall — CHANGES_REQUESTED** [review]
Just some minor wording changes please.

**Comment by naveenbalachandar** [comment]
@alanheppenstall I have made the wording changes. Could you please take another look?

**Review by alanheppenstall — APPROVED** [review]

**Pull request merged**